app/static/src/app/components/WodinSession.vue

```diff
@@ -21,36 +31,82 @@ export default defineComponent({
         defaultLanguage: String
     },
     components: {
+        SessionInitialiseModal,
         RouterView
     },
     setup(props) {
         const store = useStore();
-        const initialised = computed(() => !!(store.state.appName && store.state.baseUrl && store.state.appsPath));
+
+        const path = new URL(window.location.href).pathname;
+        const isSessionsRoute = !!path.match(/\/sessions\/?$/);
+        const sessionInitialised = ref(false);
+        const appInitialised = computed(() => !!store.state.config && !!store.state.sessions.sessionsMetadata);
+        const latestSessionId: Ref<null|string> = ref(null);
+
+        // These props won't change as provided by server
+        // eslint-disable-next-line vue/no-setup-props-destructure
```

Comment:
I think this makes sense to just disable here - the rule is intended to stop you tripping yourself up by losing reactivity if the props change (because you've destructured out the values into non-reactive consts) which is usually a valid concern, but here we know that in this particular case these are top-level component props provided by the backend so they aren't going to change while the component is mounted.